src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -93,15 +95,9 @@ public String serializeSlots(Iterable<Slot> slots, String delimiter) {
         Set<String> serializedOtherSlots = new TreeSet<>();
```

Comment:
serializedVarSlots -> serializedSlots
remove  serializedOtherSlots

Reply:
This is still open, right?